components/TitleBar.vue

```diff
@@ -0,0 +1,168 @@
+<template>
+  <div class="TitleBar">
+    <div class="Background-Wrapper">
+      <div class="header-container">
+        <h2>
+          <span class="headertitle">{{ header }}</span>
+        </h2>
+        <p>
+          <span class="headersubtitle">{{ subheader }} </span>
+        </p>
+      </div>
+      <v-card class="card">
+        <h4>
+          <span class="cardheader">
+            Help us raise Bay Area COVID awareness.
+          </span>
+        </h4>
+        <p>
+          <span class="cardsubheader">
+            Have ways to improve this site? We welcome all who wish to help
+            contribute!
+          </span>
+        </p>
+        <a href="/about">
+          <v-card color="#4738AC" class="learn">
+            <h5>
+              <span class="learnheader">Learn more about the Project</span>
+            </h5>
```

Comment:
same comment as above with the span grouping.

Reply:
on latest commit